packages/web/app/components/ProcedureModal.jsx

```diff
@@ -25,77 +73,194 @@ const getEndDateTime = ({ date, startTime, endTime }) => {
   return toDateTimeString(addDays(parseISO(actualEndDateTime), 1));
 };
 
+const useProgramResponsesQuery = (patientId, procedureId) => {
+  const api = useApi();
+  return useQuery(['patient', patientId, 'programResponses'], () =>
+    api.get(`patient/${patientId}/programResponses`, { procedureId }),
```

Comment:
This is an interesting case of the api design we were discussing in dev chat. If you were extending the scope into refactoring the API, you'd probably want to do this huh?
```suggestion
    api.get(`programResponses/${procedureId}`),
```

Reply:
yes maybe - i wont in this endpoint is main patient program responses one